index.html

```diff
@@ -693,8 +693,9 @@ <h4>parseBaseProofValue</h4>
   <bdi lang="en"><code title="LATIN SMALL LETTER U">u</code></bdi>
   (<code class="codepoint">U+0075</code>
   <code class="uname">LATIN SMALL LETTER U</code>)</span>,
```

Comment:
I can't select all the lines that matter here...

"Ensure the" is a terrible way to start any of these steps. The rephrasing I did on some of these paragraphs should extend to all of them -- i.e., change `ensure` to `if`, rephrase to make the sentence an `if, then` (with an implied `then`), and if there's an `otherwise` clause, precede it with a semicolon.

I'm happy to do the heavy lifting, if you will touch all the relevant lines in your PR (even by inserting a weird bit of whitespace or gibberish).

Reply:
@TallTed Just give me a "template" of the wording you'd like in one example then I'll apply as best I can and you can review.

Reply:
Approximately —
```
If {condition}, an error MUST be raised and SHOULD convey an error type of `{error_type}`; otherwise, {other_result}
```

Reply:
Put in those changes. Please re-review.